app/boards/arm/adv360pro/Kconfig.defconfig

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright (c) 2023 The ZMK Contributors
+# SPDX-License-Identifier: MIT
+#
+
+if BOARD_ADV360PRO_LEFT
+
+config ZMK_KEYBOARD_NAME
+    default "Adv360 Pro"
+
+config ZMK_SPLIT_ROLE_CENTRAL
+    default y
+
+
+endif # BOARD_ADV360PRO_LEFT
+
+if BOARD_ADV360PRO_RIGHT
+
+config ZMK_KEYBOARD_NAME
+    default "Adv360 Pro rt"
+
+endif # BOARD_ADV360PRO_RIGHT
```

Comment:
Is there a reason to add this name?

Reply:
Sounds like this came up for Glove80 as well, and decision was to leave it: https://github.com/zmkfirmware/zmk/pull/1892#discussion_r1390106898